code/datums/gods/patrons/curses.dm

```diff
@@ -215,7 +215,7 @@
 	owner.sexcon.arousal += 1
 
 	if(owner.mob_timers["baotha_curse"])
-		if(world.time < owner.mob_timers["baotha_curse"] + rand(15,90)SECONDS)
+		if(world.time < owner.mob_timers["baotha_curse"] + rand(10,20,30,40,50,60)SECONDS)
```

Comment:
im pretty sure rand only takes 2 numbers, does this even compile?

Reply:
no it doesnt
![image](https://github.com/user-attachments/assets/ff839093-3c3d-4f8f-b9ac-f441c60b3b41)

you baboon

Reply:
what if I used pick instead ?
![image](https://github.com/user-attachments/assets/09374c18-3b72-4b7c-8e44-253396c2f7d3)

Reply:
![image](https://github.com/user-attachments/assets/25f5b86c-556a-4c58-bd17-9676481b7849)
nvm